config/locales/server.en.yml

```
@@ -4650,6 +4650,7 @@ en:
       restored: Restored
 
   reviewables:
+    already_handled: "A staff member already reviewed this post because someone else already flagged it for the same reason. Please wait for at least 24hs before flagging it again."
```

```suggestion
    already_handled: "Thanks, but we've already reviewed that post and determined it does not need to be flagged again."
```
A couple suggested changes here: we allow category moderation so it might not always be reviewed by staff. Secondly, we might want to make the 24 hours a setting and it's probably a good idea to not let them know how long they need to wait in this particular case.